config/kibana.yml

```diff
@@ -25,7 +25,11 @@
 # elasticsearch.username: user
 # elasticsearch.password: pass
 
-# If your Elasticsearch requires client certificate and key
+# SSL for outgoing requests from the Kibana Server to the browser (PEM formatted)
+# server.ssl.cert: /path/to/your/server.crt
```

Comment:
SSL for Kibana's built in webserver. These settings secure communication between user's browser and the kibana server.